### PR TITLE
Created Detail View Controller

### PR DESCRIPTION
**Pull request by bronsonmullens**
The detail view controller currently is set up to accept mock data. I altered the MapViewController to segue information into the detail view. Once we have our endpoints, we can isolate this into a segue function and pass in a city model.

### CODE REVIEW

File: labs-ios-starter/Detail/DetailViewController.swift [hunk 1]
```diff
@@ -14,7 +14,6 @@ class DetailViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        title = "City Detail"
     }
```
Comment (alext8900) [inline labs-ios-starter/Detail/DetailViewController.swift:54]: Thats very useful good Job! (just that you made the title update with what city was being viewed.)

File: labs-ios-starter/Detail/DetailViewController.swift [hunk 2]
```diff
@@ -10,10 +10,46 @@ import UIKit
 
 class DetailViewController: UIViewController {
     
+    // MARK: - Properties
+    
+    var isFavorite: Bool = false
```
Comment (alext8900) [inline labs-ios-starter/Detail/DetailViewController.swift:15]: I like this isFavorite boolean might have to tie it in with how im gonna save the favorited to an array to visualize in the favorites collection view controller.

**Review by alext8900 — APPROVED** [review]
Code looked nice and well written. Short and sweet commits. Nice job Bronson.

File: labs-ios-starter/Detail/DetailViewController.swift [hunk 3]
```diff
@@ -10,10 +10,46 @@ import UIKit
...
+    func updateWalkabilityGrade() {
+        guard let walkability = walkability else { return }
+        walkabilityScore.text = String(walkability)
+        if walkability >= 85 {
+            walkabilityScore.textColor = .systemGreen
+        } else if walkability >= 50 {
+            walkabilityScore.textColor = .systemYellow
+        } else {
+            walkabilityScore.textColor = .systemRed
+        }
```
Comment (denced) [inline labs-ios-starter/Detail/DetailViewController.swift:52]: I like the walkability function

**Review by denced — APPROVED** [review]
Good Job setting up the detail view

**Pull request merged**